MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Views in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about the Views feature of Syncfusion .NET MAUI Calendar (SfCalendar) control and more.
+platform: maui
+control: Calendar
+documentation: ug
+---
+
+
+# Multiple Calendar Views in .NET MAUI (SfCalendar)
+The `SfCalendar` control has four Calendar views to display. It can be assigned to the control by using the [View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_View) property. By default, the `Month` view is initially rendered. The current date will be displayed initially for all the Calendar views.
+
+## Month view
+The Month view displays the current month days, and usually a few days of previous and next month. By default, initially displays the current month dates and the current date is highlighted by a separate color different from the rest of the dates color in `Month` view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/month-view.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view in .NET MAUI Calendar.](images/views/maui-month-view.png)
+
+### Number Of Visible Weeks view
+The number of visible weeks in the month view can be customized by using the `NumberOfVisibleWeeks` property in the Calendar. By default, the Month view displays with the NumberOfVisibleWeeks as `6`.
+
+The following code explains how to show the Calendar month view with `NumberOfVisibleWeeks` as `4`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar"
+                      View="Month">
+                      <Calendar:SfCalendar.MonthView>
+                        <Calendar:CalendarMonthView NumberOfVisibleWeeks = 3/>
+                      </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    NumberOfVisibleWeeks = 3,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![Number of visible weeks in .NET MAUI Calendar.](images/views/maui-month-view-number-of-weeks.png)
+
+### Week number
+By setting the `ShowWeekNumber` property, it displays the week number for the current view dates in the month view. By default, the [ShowWeekNumber](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html#Syncfusion_Maui_Calendar_CalendarMonthView_ShowWeekNumber) is set to `false`. If you need to show the week number in the month view, set the `ShowWeekNumber` as true. The week numbers will be displayed based on the ISO standard.
```

Comment:
Check the below content needed?

If you need to show the week number in the month view, set the `ShowWeekNumber` as true.

Reply:
Checked the removed the unwanted content